melodic/distribution.yaml

```diff
@@ -6139,6 +6139,18 @@ repositories:
       version: master
     status: developed
   novatel_oem7_driver:
+    doc:
+      type: git
+      url: https://github.com/novatel-gbp/novatel_oem7_driver-release.git
```

Comment:
The doc block should not point to the release repository; instead, it should point to the source repository and branch (similar to the `source` section below).

Reply:
@clalancette, thank you very much. This change has been made and is now included in db1fc4e.